### ISSUE TITLE
sbt compile/inc doesn't compile with Scala trunk

### ISSUE THREAD

**Issue by skyluc** [sbt/sbt#530]
`compile/inc/Incremental.scala` uses `immutable.Set.**` which was deprecated in 2.8 and removed in 2.10.

``` scala
val newIncludes = (process flatMap dependsOnSrc) ** excluded
```

**Comment by skyluc** [comment]
I have this problem when using the 0.13 branch.

**Comment by harrah** [comment]
Fixed in eca9ab9bb1cd8c36948920e9f3aa04415819ae1b as part of branch cleanup.
